src/screens/ChangeWalletSheet.js

```diff
@@ -0,0 +1,266 @@
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
+import { useNavigation } from 'react-navigation-hooks';
+import { useDispatch } from 'react-redux';
+import { withProps } from 'recompact';
+import { ButtonPressAnimation } from '../components/animations';
+import WalletList from '../components/change-wallet/WalletList';
+import { Sheet } from '../components/sheet';
+import { Text } from '../components/text';
+import WalletTypes from '../helpers/walletTypes';
+import {
+  useAccountSettings,
+  useInitializeWallet,
+  usePrevious,
+  useWallets,
+} from '../hooks';
+import { useWalletsWithBalancesAndNames } from '../hooks/useWalletsWithBalancesAndNames';
+import {
+  addressSetSelected,
+  createAccountForWallet,
+  walletsSetSelected,
+  walletsUpdate,
+} from '../redux/wallets';
+
+import { colors } from '../styles';
+import { abbreviations, logger } from '../utils';
+import { showActionSheetWithOptions } from '../utils/actionsheet';
+import Routes from './Routes/routesNames';
+
+const walletRowHeight = 60;
+const titleHeight = 70;
+const footerHeight = 65;
+const maxListHeight = 360;
+const Title = withProps({
+  align: 'center',
+  size: 'large',
+  weight: 'bold',
+})(Text);
```

Comment:
because `<Title>` is only used once, you should just put these props directly on the `<Text>` in the render function